### PR TITLE
easeljs stageGL support

### PR DESCRIPTION
**Pull request by Fire7**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] Run `npm run lint package-name` (or `tsc` if no `tslint.json` is present).

Select one of these and delete the others:

If changing an existing definition:
- [x] Provide a URL to documentation or source code which provides context for the suggested changes: https://www.createjs.com/docs/easeljs/classes/StageGL.html
- [x] Increase the version number in the header if appropriate.
- [x] If you are making substantial changes, consider adding a `tslint.json` containing `{ "extends": "dtslint/dt.json" }`.

### CODE REVIEW

**Comment by Fire7** [comment]
@evilangelist

File: types/easeljs/index.d.ts [hunk 1]
```diff
@@ -924,6 +941,52 @@ declare namespace createjs {
...
+        vocalDebug: boolean;
+
+        // methods
+        static buildUVRects(spritesheet: SpriteSheet, target?: number, onlyTarget?: boolean): Object;
+        cacheDraw(target: DisplayObject, filters: Filter[], manager: BitmapCache): boolean;
+        getBaseTexture(w?: number, h?: number): WebGLTexture | null;
+        getFilterShader(filter: Filter | Object): WebGLProgram;
+        getRenderBufferTexture (w: number, h: number): WebGLTexture;
+        getTargetRenderTexture (target: DisplayObject, w: number, h: number): Object;
+        isWebGLActive(ctx: CanvasRenderingContext2D): boolean;
```
Comment (evilangelist) [inline types/easeljs/index.d.ts]: According to the docs, this is static

File: types/easeljs/index.d.ts [hunk 2]
```diff
@@ -924,6 +941,52 @@ declare namespace createjs {
 
     }
 
+    export class StageGL extends Stage {
+        constructor(canvas: HTMLCanvasElement | string | Object);
```
Comment (evilangelist) [inline types/easeljs/index.d.ts]: Would be nice to define an interface for the options Object with the applicable properties. It also takes a second optional arg of options.

**Review by evilangelist — CHANGES_REQUESTED** [review]
Looks reasonable, just a couple of suggested changes. I haven't worked with this library in years so I could only use the docs as reference.

File: types/easeljs/index.d.ts (hunk 2 above)
Comment (Fire7) [reply types/easeljs/index.d.ts]: forget about it, fixed)

File: types/easeljs/index.d.ts (hunk 1 above)
Comment (Fire7) [reply types/easeljs/index.d.ts]: My mistake..

**Comment by Fire7** [comment]
@evilangelist Thanks for review, I have fixed your comments.

**Review by evilangelist — APPROVED** [review]
Looks good, thanks for the updates.

**Pull request merged**